zuul-core/src/main/java/com/netflix/netty/common/proxyprotocol/ElbProxyProtocolChannelHandler.java

```diff
@@ -16,142 +16,43 @@
 
 package com.netflix.netty.common.proxyprotocol;
 
-import com.google.common.net.InetAddresses;
-import com.netflix.netty.common.SourceAddressChannelHandler;
-import io.netty.channel.Channel;
-import io.netty.channel.ChannelFutureListener;
+import io.netty.buffer.ByteBuf;
 import io.netty.channel.ChannelHandlerContext;
 import io.netty.channel.ChannelInboundHandlerAdapter;
 import io.netty.channel.ChannelPipeline;
-import io.netty.handler.codec.haproxy.HAProxyMessage;
-import io.netty.handler.codec.haproxy.HAProxyProtocolVersion;
-import io.netty.util.AttributeKey;
-
-import java.net.InetSocketAddress;
-import java.net.SocketAddress;
-
-import io.netty.util.ReferenceCounted;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import io.netty.handler.codec.ProtocolDetectionState;
+import io.netty.handler.codec.haproxy.HAProxyMessageDecoder;
 
 /**
- * Copies any decoded HAProxyMessage into the channel attributes, and doesn't pass
- * it any further along the pipeline.
- *
- * Use in conjunction with OptionalHAProxyMessageDecoder if ProxyProtocol is enabled on the ELB.
- *
- * User: michaels@netflix.com
- * Date: 3/24/16
- * Time: 11:59 AM
+ * Decides if we need to decode a HAProxyMessage. If so, adds the decoder followed by the handler.
+ * Else, removes itself from the pipeline.
+ * User: arghac@netflix.com Date: 02/19/2020
```

Comment:
I would highly recommend not putting your name here.  It suggests the code is owned by the individual, rather than the team/community.

Reply:
Agreed.

Reply:
Disagree with @carl-mastrangelo on this. I think the name helps set a pointer to who has context on this code. Your call.